botstrap.py

```diff
@@ -15,6 +15,12 @@
 BOT_TOKEN = os.getenv("BOT_TOKEN", None)
 GUILD_ID = os.getenv("GUILD_ID", None)
 
+COMMUNITY_FEATURE = "COMMUNITY"
+PYTHON_HELP_CHANNEL_NAME = _Channels.__fields__["python_help"].name
+ANNOUNCEMENTS_CHANNEL_NAME = _Channels.__fields__["announcements"].name
+RULES_CHANNEL_NAME = _Channels.__fields__["rules"].name
```

Comment:
Maybe I'm missing something, but you're using the channel name to access the... channel name? seems circular.

Reply:
You're not missing anything.
It's because I had imagined something different when I was starting up.

But they actually map the same value, so it's pointless.

I'll fix it

Reply:
6a782da04193b825220de2a9cefab1983351e134